frontend/src/containers/Signup/Signup.tsx

```diff
@@ -37,23 +37,31 @@ const SignupForm = () => {
       }),
       headers: { 'Content-Type': 'application/json' },
     });
+
     const resdata = await res.json();
+
     if (res.status === 200) {
-      router.replace('/login');
+      // Call the verification API after successful signup
+      await fetch(`${process.env.NEXT_PUBLIC_API_URL}/verify-email/`, {
+        method: 'POST',
+        body: JSON.stringify({ email: data.email }),
+        headers: { 'Content-Type': 'application/json' },
+      });
+
+      // Redirect to verify-email page
+      router.replace(`/verify-email?email=${encodeURIComponent(data.email)}`);
```

Comment:
This part is incorrect. You just have to send GET request to `http://localhost:8000/api/v1/request-email-verify/` endpoint with header as `Authorization: Bearer <access_token>` as mentioned in the [issue](https://github.com/djangoindia/djangoindia.org/issues/284). 

On doing this, backend will automatically send the mail to the user.

Except calling the above API after the user is created manually (not with google sign in), there will be no change in this page.